.github/workflows/tests.yml

```diff
@@ -113,19 +113,19 @@ jobs:
                         --skipped-or-failed-formulae=${{ steps.brew-test-bot-formulae.outputs.skipped_or_failed_formulae }}
 
       - name: Output brew test-bot failures
-        if: runner.os == 'macOS'
         run: |
           cat steps_output.txt
           rm steps_output.txt
 
       - name: Output brew bottle output
-        if: runner.os == 'macOS'
         run: |
           cat bottle_output.txt
           rm bottle_output.txt
 
       - run: brew test-bot --only-cleanup-after
 
+      - run: rm -rvf *
```

Comment:
Can/should this be more specific?

Reply:
I don't feel strongly either way, but I've made it specific to just the bottle tarball.